composer.json

```diff
@@ -23,18 +23,19 @@
         "docs": "https://tenancy.dev"
     },
     "require": {
-        "php": "^7.2",
+        "php": "^7.3",
         "phpseclib/phpseclib": "~2.0",
         "doctrine/dbal": "~2.5",
         "ramsey/uuid": "^4.0",
-        "laravel/framework": "^7.0"
+        "laravel/framework": "^8.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.0",
-        "symfony/dom-crawler": "~3.1",
-        "laravel/laravel": "^7.0",
+        "fzaninotto/faker": "^1.8",
+        "laravel/laravel": "^8.0",
+        "laravel/legacy-factories": "^1.0",
```

Comment:
You add Legacy Factories, but you fixed a factory? Would it be worth it to fix all factories?

Reply:
👍 I can go in and update all factories, no problem.

Reply:
Would really appreciate it ❤️